api/web3server.go

```diff
@@ -744,18 +707,15 @@ func (svr *Web3Server) getTransactionByBlockHashAndIndex(in interface{}) (interf
 	if err != nil {
 		return nil, err
 	}
-	actionInfos, err := svr.coreService.ActionsByBlock(util.Remove0xPrefix(blkHash), idx, 1)
+	blkHash = util.Remove0xPrefix(blkHash)
+	selps, receipts, err := svr.coreService.ActionsInBlockByHash(blkHash)
+	if errors.Cause(err) == ErrNotFound || idx >= uint64(len(receipts)) || len(receipts) == 0 {
```

Comment:
can we use existing
```
st, ok := status.FromError(err)
		if ok && st.Code() == codes.NotFound {
			return nil, nil
		}
```
instead of introducing a new `ErrNotFound`?

Reply:
no. codes.NotFound will be removed in the coreservice